raiden/transfer/node.py

```diff
@@ -604,6 +606,46 @@ def handle_action_change_node_network_state(
     return TransitionResult(chain_state, events)
 
 
+def handle_action_set_token_network_fee_schedule(
+    chain_state: ChainState, state_change: ActionTokenNetworkSetFeeSchedule
+) -> TransitionResult[ChainState]:
+    events: List[Event] = []
+
+    token_network_address = state_change.token_network_address
+    fee_schedule = state_change.fee_schedule
+    token_network_registries = chain_state.identifiers_to_tokennetworkregistries
+    token_network_registry_state = token_network_registries[state_change.registry_address]
+
+    if token_network_registry_state is None:
+        return TransitionResult(chain_state, events)
+
+    ids_to_tokens = token_network_registry_state.tokennetworkaddresses_to_tokennetworks
+    token_network_state = ids_to_tokens[token_network_address]
```

Comment:
The reason I avoided the views function and tried to manipulate the dictionary directly was because there is the intent of setting the updated values into the `tokennetworkaddresses_to_tokennetworks`.

Using the functions from the views module would just return a copy of the variables and consequently make it impossible of persisting the new values, right?

Reply:
You should **only** update state machine values through a state change. What you're trying to do is a no-go.